docs/arf.md

```diff
@@ -198,14 +198,14 @@ User\'s perspective. It spans from acquiring a valid Wallet Instance to
 the process of identifying and authenticating themselves for an online
 service. The focus here is on a practical remote same-device flow (as
 detailed in [section 4.2.2](#422-attestation-presentation-flows) and [4.2.3](#423-mobile-apps-and-web-browsers)). In this context, a User utilises a
-single device for both securing their session and accessing the service,
+single device [**What about scenarios where the wallet device is used for multi-factor euthentication for a service being accessed on another device? This is a valid use case as not everything will be done on a mobile phone**]for both securing their session and accessing the service,
```

Comment:
Remote cross-device presentation scenarios are discussed in chapter 4. In fact, section 2.1 should not talk about the distinction between same-device and cross-device. This paragraph will be removed.